lib/middleman-hashicorp/reshape/reshape.js

```diff
@@ -0,0 +1,20 @@
+const reshape = require('reshape')
+const preactPreset = require('babel-preset-preact')
+
+require('babel-register')({ presets: [preactPreset] })
+
+const components = require(`${process.cwd()}/assets/reshape`)
```

Comment:
I feel like we might as well move the `registerComponents` call into this file and just have projects export a config object for it

Reply:
Another thing we could potentially do is allow the path to this file to be configurable using something like an environment variable that we could drop into [this line](https://github.com/carrot/middleman-hashicorp/pull/3/files#diff-7d0c8ffd4e7a176a44999e9f6ef11080R19) through ruby. This would be I think as close as we can get to allowing the config to pass through ruby.